prepare-image.sh

```diff
@@ -32,6 +32,12 @@ if [[ $INSTALL_TYPE == "source" ]]; then
 
     python3 -c 'import os; import sys; import jinja2; sys.stdout.write(jinja2.Template(sys.stdin.read()).render(env=os.environ, path=os.path))' < "${IRONIC_PKG_LIST}" > "${IRONIC_PKG_LIST_FINAL}"
 
+    if [ -n $SUSHY_SOURCE ]; then
+        UPPER_CONSTRAINTS_FILE="/tmp/sushy-constraints.txt"
+        curl -L https://releases.openstack.org/constraints/upper/master -o $UPPER_CONSTRAINTS_FILE
```

Comment:
The existing code accounts for an UPPER_CONSTRAINTS_FILE provided by a user. Maybe do it like this:

```bash
curl -L ${UPPER_CONSTRAINTS_FILE:-"https://releases.openstack.org/constraints/upper/master"} -o /tmp/sushy-constraints.txt
UPPER_CONSTRAINTS_FILE="/tmp/sushy-constraints.txt"
```

Reply:
yeah, looks like a good solution for that, going to update the change
thanks!